core/src/test/scala/unit/kafka/utils/JaasTestUtils.scala

```diff
@@ -42,15 +40,38 @@ object JaasTestUtils {
     }
   }
 
-  case class JaasSection(contextName: String,
-                         moduleName: String,
-                         debug: Boolean,
-                         entries: Map[String, String]) {
+  case class PlainLoginModule(username: String,
+                              password: String,
+                              debug: Boolean = false,
+                              validUsers: Option[Map[String, String]] = None) {
+    def toJaasModule: JaasModule = {
+      JaasModule(
+        "org.apache.kafka.common.security.plain.PlainLoginModule",
+        debug = debug,
+        entries = Map(
+          "username" -> username,
+          "password" -> password
+        ) ++ validUsers.getOrElse(Map.empty).map(up => ("user_" + up._1 -> up._2))
```

Comment:
It's more readable if we write this as `map { case (user, password) => (s"user_$user" -> password }` or something like that.